system/LINK/bin/fix_auth

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# description: ManageIQ fix_auth launcher
+#
+
+if [ -d /var/www/miq/vmdb ] ; then
+  cd /var/www/miq/vmdb
+else
+  # remove symbolic links to script
+  SCRIPT=$(readlink "$0")
+  # dir name
+  SCRIPTPATH=$(dirname "${SCRIPT:-$0}")
+  # rails root
+  cd "${SCRIPTPATH}/../../../vmdb"
```

Comment:
Sorry, I didn't make myself clear. I don't understand why a developer would run this wrapper script vs just running fix_auth.rb directly. In other words, what value is there to the else condition?